lib/notify_delivery_method.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "notifications/client"
+
+class NotifyDeliveryMethod
+  def initialize(mail); end
```

Comment:
Does this need to exist if it doesn't do anything?

Reply:
Ah yes - sorry this was a hangover from trying to figure out how to get it to work with the mailer.